warehouse/archive/archiver.go

```diff
@@ -510,3 +512,79 @@ func (a *Archiver) deleteLoadFileRecords(
 
 	return nil
 }
+
+func (a *Archiver) Delete(ctx context.Context) error {
+	a.log.Infof(`[Archiver]: Started deleting for warehouse`)
```

Comment:
We can get rid of the `[Archiver]:` in logs.
```suggestion
	a.log.Infon(`Started deleting upload jobs`)
```